verify/src/main/java/com/twilio/verify/TwilioVerifyException.kt

```diff
@@ -83,3 +85,22 @@ class TwilioVerifyException(
     AuthenticationTokenError("{$AUTHENTICATION_TOKEN_CODE} Exception while generating token")
   }
 }
+
+/**
+ * Exception types returned as cause for a [TwilioVerifyException] and [InputError] code on validation errors.
+ *
+ * @param message associated message of the exception.
+ */
+sealed class InputException(override val message: String) : Exception(message)
+object ExpiredChallengeException : InputException("Expired challenge can not be updated")
```

Comment:
should `message` be a string resource instead?

Reply:
We don't need localization for the message, the idea is to explain the reason to the developer.